lib/json_utils.h

```diff
@@ -0,0 +1,66 @@
+/*  Copyright (C) 2019  Red Hat, Inc.
+ *
+ *  satyr is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  satyr is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <json.h>
+#include <stdbool.h>
+
+/* Checks whether json value is of the right type and sets the error_message if
+ * not. The "name" argument is used in the error message. */
```

Comment:
Could you put the comment terminator on a separate line?

Reply:
The code was copied, didn’t really care about whatever demons were holed up there.

Will do.